src/Components/Game/Game.js

```diff
@@ -1,154 +1,148 @@
-                {id: 6, visibility: 'visible'},
-                {id: 7, visibility: 'visible'},
-                {id: 8, visibility: 'visible'},
-                {id: 9, visibility: 'visible'},
-            ],
-            bombID: null,
-            pointMultiplier:0,
-            lastScore: []
-        }
-
-        //this.lastScore = [];
-
-
-        this.handleResetClick = this.handleResetClick.bind(this);
-        this.handleChangeScoreClick = this.handleChangeScoreClick.bind(this);
-        this.onHideItem = this.onHideItem.bind(this);
-        this.resetPoints = this.resetPoints.bind(this);
-        this.newGame = this.newGame.bind(this)
-        this.updateLastScore = this.updateLastScore.bind(this)
+import React from "react";
+import ResetBtn from "./ResetBtn";
+import PuzzleItem from "./PuzzleItem";
+import Score from "./Score";
+
+import Timer from "../Timer/Timer";
+import LastScore from "./LastScore";
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      score: 0,
+      items: [
+        { id: 1, visibility: "visible" },
+        { id: 2, visibility: "visible" },
+        { id: 3, visibility: "visible" },
+        { id: 4, visibility: "visible" },
+        { id: 5, visibility: "visible" },
+        { id: 6, visibility: "visible" },
+        { id: 7, visibility: "visible" },
+        { id: 8, visibility: "visible" },
+        { id: 9, visibility: "visible" },
+      ],
+      bombID: null,
+      pointMultiplier: 0,
+      lastScore: [],
+    };
+
+    this.handleResetClick = this.handleResetClick.bind(this);
+    this.handleChangeScoreClick = this.handleChangeScoreClick.bind(this);
+    this.onHideItem = this.onHideItem.bind(this);
+    this.resetPoints = this.resetPoints.bind(this);
+    this.newGame = this.newGame.bind(this);
+    this.updateLastScore = this.updateLastScore.bind(this);
+  }
+
+  componentDidMount() {
+    this.getRandomBomb();
+  }
+
+  resetPoints() {
+    this.setState({
+      score: 0,
+      pointMultiplier: 0,
+    });
+  }
+
+  updateLastScore() {
+    let lastScoreArr = this.state.lastScore;
+    if (lastScoreArr.length > 9) {
+      this.setState({
+        lastScore: [],
+      });
     }
 
-    componentDidMount(){
-        this.getRandomBomb()
-    }
-
-    resetPoints(){
-        this.setState({
-            score: 0,
-            pointMultiplier: 0
-        })
-    }
-
-    updateLastScore(){
-        let lastScoreArr = this.state.lastScore;
-        if(lastScoreArr.length > 9 ){
-            this.setState({
-                lastScore:[]
-            })
-        }
-        // this.lastScore.push(this.state.score);
-        this.setState(state =>{
-            const lastScore = [...state.lastScore, state.score]
-
-            return {lastScore}
-        })
-    }
-
-    handleResetClick(){
-        this.getRandomBomb()
-        this.setState({
-            pointMultiplier: 0
-        });
-        this.setState(
```

Comment:
Invalid usage of setState. Instead of passing an object with the key and value, you're passing the result of `.map` function. In its turn `.map` function does not return anything in your case. It should return a new object on every iteration. It should look like in the PR:

<img width="337" alt="Screen Shot 2020-07-04 at 19 28 22" src="https://user-images.githubusercontent.com/17337276/86513495-87899280-be2c-11ea-909f-64c3dc22cb3d.png">